spec/_attachments/interface-spec-changelog.md

```diff
@@ -11,6 +11,7 @@
 * Set the maximum depth of a delegation in a read_state response/certified variable certificate to 1.
 * Canister version is guaranteed to increase if the canister's running status changes.
 * Calls to frozen canisters are rejected with `SYS_TRANSIENT` instead of `CANISTER_ERROR`.
+* Add API boundary nodes into the certified state tree and new HTTP endpoints for retrieving them.
```

Comment:
```suggestion
* Add API boundary nodes information into the certified state tree.
```

Reply:
We reuse the existing read_state endpoints (we just change what can be requested through them).